db/seeds.rb

```diff
@@ -5,3 +5,39 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+if Rails.env == 'development'
+  Product.destroy_all
+  Ingredient.destroy_all
+
+  ['Chicken', 'Tofu', 'Salmon'].each do |ingredient|
+    Ingredient.create(name: ingredient, price: 25 + rand(10), available: true)
+  end
+
+  ['Bibimbap', 'Funchöza', 'Hoe-deopbap', 'kimchi-jjigae', 'Korean tacos'].each do |dish|
+    prod = Product.create(name: dish,
+                             price: 89,
+                             description: 'Tasty food',
```

Comment:
Just for the sake of displaying a dish description nicely, it would be good if it was longer so we can take that in to account when styling.

Reply:
ok, will do that